airflow/opt/providers/etna/setup.py

```diff
@@ -25,6 +25,7 @@
         "etna.dags",
         "etna.etls",
         "etna.metrics",
+        "etna.plugins"
```

Comment:
Required so that the plugins are also installed.

Reply:
Cool, that works.  I seem to recall there was also a way of specifying a plugins entrypoint specifically?  But if this works it works.  Python has a history for a rather divergent and inconsistent ecosystem for its packaging system.